stock_product_pack/tests/test_stock_product_pack.py

```diff
@@ -122,10 +81,13 @@ def test_compute_quantities_dict(self):
             }
         )
         picking.action_confirm()
-        self.assertEqual(self.pack_dc.virtual_available, 5)
-        self.assertEqual(self.pack_dc.qty_available, 0)
+        company = picking.company_id
+        pack = self.pack_dc.with_context(company_id=company.id)
```

Comment:
But the key to trigger the `depends_context` is `force_company`. Try it and let's see if the test succeds without the `invalidate_cache` trick

Reply:
I have tried passing `force_company=company.id` ans still no luck :disappointed:. Was this what you meant?

Reply:
Yes. Then assigning the context here is useless. Just `invalidate_cache` por this test

Reply:
Ok. Leaving this asis, then. Thank you anyway for the idea!

Reply:
What I mean is that this context is not very useful. So just add the line with `invalidate_cache`

Reply:
Ok. Changes made. Thank you.